src/index.js

```diff
@@ -47,16 +47,12 @@ function renderToStream(componentPath, data, vueOptions, GlobalOptions): Promise
             .then(app => {
                 const vueStream = vueServerRenderer.renderToStream(app.app);
                 let htmlStream;
+                const htmlStringStart = app.template.start + app.head + app.template.middle;
+                const htmlStringEnd = app.script + app.template.end;
 
-                if (!GlobalOptions.pure) {
```

Comment:
My fear of removing this is that large streams will get blocked. Instead of multiple data buffer events it will be one.